solutions/bronze/usaco-1013.mdx

```diff
@@ -162,3 +165,61 @@ int main() {
 	for (int t : cur) cout << t + 1 << "\n";
 }
 ```
+
+</CPPSection>
+<PySection>
+
+```py
+from typing import List
+
+
+def reverse_segment(cows: List[int], start: int, end: int):
+	while start < end:
+		cows[start], cows[end] = cows[end], cows[start]
+		start += 1
+		end -= 1
+
+
+def find_cycle_length(cows: List[int], a1: int, a2: int, b1: int, b2: int) -> int:
```

Comment:
this function isn't even used what

Reply:
oops, that was from another approach. I got rid of it .